add '(or subset)' after config (#1347)
huggingface/hub-docs: datasetcard.md

- {subtask_1}  # Example: multi-class-image-classification
paperswithcode_id: {paperswithcode_id}  # Dataset id on PapersWithCode (from the URL). Example for SQuAD: squad
configs:  # Optional. This can be used to pass additional parameters to the dataset loader, such as `data_files`, `data_dir`, and any builder-specific parameters  

> - config_name: {config_name_0}  # Name of the configuration (or subset). Example: default

Reviewer: ```suggestion
- config_name: {config_name_0}  # Name of the dataset subset, if applicable. Example: default
```

you mean like this @lhoestq?

    #       dtype: string
    #     - name: answer_start
    #       dtype: int32
  config_name: {config_name}  # Name of the configuration (or subset). Example for glue: sst2
  splits:
    - name: {split_name_0}                  # Example: train
      num_bytes: {split_num_bytes_0}        # Example for SQuAD: 79317110
      num_examples: {split_num_examples_0}  # Example for SQuAD: 87599
  download_size: {dataset_download_size}   # Example for SQuAD: 35142551
  dataset_size: {dataset_size}             # Example for SQuAD: 89789763

> # It can also be a list of multiple configurations (or subsets):

Reviewer: here i think it's fine this way

Reviewer: or this:
> subsets (also called "configurations")

Author: https://github.com/huggingface/hub-docs/pull/1348
